Application/EdFi.Ods.Features/ChangeQueries/Repositories/TrackedChangesQueryTemplatePreparerBase.cs

```diff
@@ -40,14 +40,9 @@ public TrackedChangesQueryTemplates PrepareQueryTemplates(IQueryParameters query
             // Finalize the query builders, cloning them and then applying request-specific parameters
             var dataQueryBuilder = FinalizeDataQueryBuilder(queryBuilder.Clone(), queryParameters, resource);
 
-            // Prepare a count query, if requested
-            var countQueryBuilder = queryParameters.TotalCount
-                ? FinalizeCountQueryBuilder(queryBuilder.Clone(), queryParameters)
```

Comment:
Method `FinalizeCountQueryBuilder` isn't used anymore, can we remove it?

Reply:
Removed.